community-containers/nextcloud-exporter/nextcloud-exporter.json

```diff
@@ -0,0 +1,37 @@
+{
+    "aio_services_v1": [
+        {
+            "container_name": "nextcloud-aio-nextcloud-exporter",
+            "display_name": "Prometheus Nextcloud Exporter",
+            "documentation": "https://github.com/nextcloud/all-in-one/tree/main/community-containers/nextcloud-exporter",
+            "image": "ghcr.io/xperimental/nextcloud-exporter",
+            "image_tag": "0.8.0",
```

Comment:
Just to mention it: If you pin a tag, you will need to update the tag when a new comes out. Using `latest` would solve this even though it is of course usually not recommended. How I solve this for other containers is to have a dedicated tag like for example `v0` which only increases if there are some breaking changes so that I would need to increase to v1.